2016/presentations/splash/text.md

```diff
@@ -0,0 +1,83 @@
+
+Title of Presentation
+=====================
+
+[Splash](http://splash.readthedocs.io/en/stable/) is an open source scriptable headless browser with 
+HTTP API written in Python 3 with Twisted & PyQt. I’d like to present the project and discuss 
+couple of social and technical phenomena that I noticed while taking part in development of 
+application. I’d like to discuss how application is structured, what libraries we used and
+how they served us, and finally discuss how open source works for us and how I perceive it.
+
+My presentation will have two parts, one technical and other "social/psychological". In the first 
+one I'll outline application architecture. In the second one I'll try to discuss some general
+issues around open source that I noticed.
+
+What is Splash and how it works?
+================================
+
+Splash is a headless browser based on WebKit engine. Its main job is rendering HTML and JavaScript.
+You can interact with Splash by sending HTTP requests with some parameters. For example you can make
+request to /render.png endpoint with `url` parameter and it will send request to given url, load
```

Comment:
"request to" → "request to the"
"with `url` paramater" → "with a `url` parameter"